index.html

```diff
@@ -4072,13 +4072,16 @@ <h2>Elements</h2>
 <p>A <dfn>node id map</dfn> is weak map between nodes and their
 corresponding <a>WebDriver node id</a>.
 
+<p>A [=Window=] has a [=set=] <dfn>seen node ids</dfn>, which is
+initially empty.
+
 <p>To <dfn>get a node</dfn> given <var>session</var>
-and <var>reference</var>:
+<a>browsing context</a>, and <var>reference</var>:
 <ol>
   <li>Let <var>browsing context group node map</var>
   be <var>session</var>'s <a>browsing context group node map</a>.
 
-  <li>Let <var>browsing context group</var> be <a>current browsing
+  <li>Let <var>browsing context group</var> be <a>browsing
   context</a>'s <a>browsing context group</a>.
```

Comment:
```suggestion
  <li>Let <var>browsing context group</var> be <var>browsing
  context</var>'s <a>browsing context group</a>.
```

Reply:
Maybe I misunderstand the syntax, and the `<var>some_var</var>` should be declared only once, and used as `<a>some_var</a>`. If so, please decline the suggestions.